flowtools/fcs_scatterplot/getDensityPlots.xml

```diff
@@ -1,15 +1,15 @@
-<tool id="gen_density_plots" name="Generate scatter plots" version="1.1">
+<tool id="gen_density_plots" name="Generate scatter plots" version="1.0+galaxy0">
   <description>for any given markers in FCS file</description>
   <requirements>
-    <requirement type="package" version="2.0.2">ig-flowtools</requirement>
+    <requirement type="package" version="1.10.0">bioconductor-flowdensity</requirement>
   </requirements>
   <stdio>
     <exit_code range="10" level="fatal" description="Please provide a comma separated list of channels to plot" />
```

Comment:
Please cover all exit code ranges >0 (for examples, see recent commits to master), for the rest of the numbers. For all new ranges, don't specify a description so that the tools stderr is pasted in the UI by Galaxy.